packages/dcos-integration-test/extra/test_metrics.py

```diff
@@ -192,6 +193,72 @@ def _nginx_vts_measurement_basename(name: str) -> str:
     return '_'.join(name.split('_')[:3])
 
 
+def test_metrics_master_adminrouter_nginx_drop_requests_seconds(dcos_api_session):
+    """
+    nginx_vts_*_request_seconds* metrics are not present.
+    """
+    # Make request to a fine-grained metrics annotated upstream of
+    # Admin Router (IAM in this case).
+    dcos_api_session.get('/acs/api/v1/auth/jwks')
+    node = dcos_api_session.masters[0]
```

Comment:
What I worry about in this test, and the already existing test (written by myself) is that in a multi-master setup the `get` request above is made to another node than `master[0]` which we use to test the metrics emitted for the former.